PhysicsTools/PatAlgos/src/classes_def.xml

```diff
@@ -0,0 +1,16 @@
+<lcgdict>
+
+ <class name="pat::HcalDepthEnergyFractionProd">
+   <field name="hcalDepthEnergyFraction_" transient="true"/>
+ </class>
+ <class name="std::vector<pat::HcalDepthEnergyFractionProd>"/>
+ <class name="edm::Wrapper<std::vector<pat::HcalDepthEnergyFractionProd> >"/>
+ <class name="edm::ValueMap<pat::HcalDepthEnergyFractionProd>" />
+ <class name="edm::Wrapper<edm::ValueMap<pat::HcalDepthEnergyFractionProd> >"/>
+
+ <ioread sourceClass = "HcalDepthEnergyFractionProd" version="[1-]" targetClass="HcalDepthEnergyFractionProd" source="std::vector<uint8_t> hcalDepthEnergyFractionI_" target="hcalDepthEnergyFraction_">
+ <![CDATA[ newObj->initUint8();
+ ]]>
```

Comment:
this looks backwards: the goal should be to initialize fraction float vector from the int, not the other way around.

Reply:
> the goal should be to initialize fraction float vector from the int,

I agree and I think thta's what I am doing, but let me know if this still needs to be changed.

Reply:
Never mind. I understand what you mean now and I now use a new initFloatVector() here that converts the stored uint8 to float.